File: java/euler/src/main/java/io/angelsanchezt/euler/Problem001.java
package io.angelsanchezt.euler;


/* 
 * Solution to Project Euler problem 1
 * 
 * https://github.com/AngelSanchezT/Project-Euler-solutions
 */
public class Problem001 {

	public static int solution(int len) {
		int sum = 0;
		for (int i = 0; i < len; i++) {
			if (i % 3 == 0 || i % 5 == 0)
				sum += i;
		}
		return sum;
	}

}

File: Javascript/problems/problem001.js
function problem001(target) {
    let sum = 0;
    for(let i = 1; i < target; i ++) {
        if((i % 3 === 0) || (i % 5 === 0)) {
            sum += i;
        }
    }
    return sum;
}

module.exports = problem001;